src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -78,10 +78,20 @@ public override async Task<string> ScheduleNewOrchestrationInstanceAsync(
     {
         Check.NotEntity(this.options.EnableEntitySupport, options?.InstanceId);
 
+        var version = string.Empty;
```

Comment:
code style nit: we try to avoid using `var` in this repo.
```suggestion
        string version = string.Empty;
```

Reply:
Done, I'll keep this in mind for future PRs :)